libminifi/include/core/Property.h

```diff
@@ -467,6 +468,16 @@ class ConstrainedProperty : public std::enable_shared_from_this<ConstrainedPrope
   friend class PropertyBuilder;
 };
 
+struct ConfigurationProperty {
+  explicit ConfigurationProperty(std::string_view name_, gsl::not_null<PropertyValidator*> validator_ = gsl::make_not_null(StandardValidators::get().VALID_VALIDATOR.get()))
+    : name(name_),
+      validator(validator_) {
```

Comment:
According to the [google style guide](https://google.github.io/styleguide/cppguide.html#Variable_Names) in case of structs we shouldn't use `_` suffixes for members, I used them on the local variables for distinguishing them instead.

Reply:
The style guide says not to use `_` for parameters, either, so we are deviating from it in either case.  I think using `_` for the struct members is the smaller deviation, and fits the rest of our code more.  But if you like it better this way, that's fine, too.

Reply:
Removed the suffixes altogether as it is not needed in the initializer list.